src/OrchardCore.Modules/OrchardCore.Users/Views/TwoFactorAuthentication/Index.cshtml

```diff
@@ -8,7 +8,9 @@
 @inject ITwoFactorAuthenticationHandlerCoordinator TwoFactorAuthenticationHandlerCoordinator
 @{
     RenderTitleSegments(T["Two-Factor Authentication"]);
+    ViewLayout = "Layout__TwoFactor";
 }
+<script asp-name="font-awesome" at="Head" version="6"></script>
```

Comment:
`Layout-Login` has this at the `Foot`, and here that seems to be working fine too. However, both cause this with The Blog Theme:

![image](https://github.com/OrchardCMS/OrchardCore/assets/1976647/d848f6f3-0974-433d-a11d-040699f95632)

Reply:
Not sure what is causing this. Any idea? I made sure all the layouts are injecting the FA scripts in the footer. I do not want to open another can of warms here with CSS

Reply:
I think here too the reason is that we are not overriding the Layout-Login and Layout-TwoFactor which is causing this rendering issue.